testsuite/pytests/test_compartmental_model.py

```diff
@@ -959,8 +959,12 @@ def test_continuerun(self, dt=0.1):
         m_neat = nest.Create("multimeter", 1, {"record_from": recordables, "interval": 1.0})
         nest.Connect(m_neat, n_neat)
 
-        nest.Simulate(12.0)
-        nest.Simulate(88.0)
+        # test the case where we continue a run without resetting the neural state
```

Comment:
Should repeated calls to Simulate() still be tested? Or should this be forbidden?

Reply:
I don't know how this code with the two `Simulate()` calls got implemented in the test, my version (and I've reverted the implementation to this now) is:
"""
        nest.Prepare()
        nest.Run(12.0)
        nest.Run(88.0)
        nest.Cleanup()
 """
It tests whether continuing the simulation without resetting the internal state of the model works as expected. Or is this not a proper use of these functions? As far as I'm aware, two `Simulate()` calls don't have the same behaviour.